### PR TITLE
Implement Travis and phpunit basic configuration

### PR DESCRIPTION
**Pull request by pradzikowski**
Its just basic testsuite and travis config.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         syntaxCheck="false"
+         bootstrap="vendor/autoload.php"
+>
+    <testsuites>
+        <testsuite name="ValueObject testsuite">
+            <directory>./tests/</directory>
```
Comment (adeptofvoltron) [inline phpunit.xml.dist]: actually the dir name We created is 'test' without "s" in the end.

change dir name and namespace in composer.json ...or change directory name in phpunit.xml.dist

I think first option is better

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adeptofvoltron) [inline phpunit.xml.dist:1]: ifyou created .dist file it would be nice to add 'local file' to gitignore

File: phpunit.xml.dist (hunk 1 above)
Comment (pradzikowski) [reply phpunit.xml.dist]: Right i forgot this case

File: phpunit.xml.dist (hunk 2 above)
Comment (pradzikowski) [reply phpunit.xml.dist:1]: done

File: phpunit.xml.dist (hunk 1 above)
Comment (pradzikowski) [reply phpunit.xml.dist]: done

**Review by adeptofvoltron — COMMENTED** [review]
image

**Review by adeptofvoltron — APPROVED** [review]

**Comment by pradzikowski** [comment]
Next thing which, has been forgoten  :P

**Pull request merged**